image/image.go

```diff
@@ -121,12 +121,37 @@ func localSnaps(tsto *ToolingStore, opts *Options) (*localInfos, error) {
 	}, nil
 }
 
+func validateNoParallelSnapInstances(snaps []string) error {
+	for _, snapName := range snaps {
+		_, instanceKey := snap.SplitInstanceName(snapName)
+		if instanceKey != "" {
+			return fmt.Errorf("cannot use snap %q, parallel snap instances are unsupported",
```

Comment:
This is a slightly unusual line wrapping. Its more common to put snapName into the same line (we don't care about 80char lines etc).